components/Table/style.css

```diff
@@ -51,8 +58,17 @@
 }
 .table-row__cell {
   flex: 1;
-  padding-left: 5px;
+  padding-left: 10px;
+}
+.material-icons[name='arrow'][asc='true'] {
```

Comment:
This selector(and above) looks overhead
Let's use class for this icon and check `asc` attribute